sdk/containerregistry/Azure.Containers.ContainerRegistry/Directory.Build.props

```diff
@@ -0,0 +1,3 @@
+﻿<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory).., Directory.Build.props))\Directory.Build.props" />
```

Comment:
do you know why this is needed? I see it in the template and in FR too.
I went to FR and deleted the file and was able to build and run all tests.. so idk

Reply:
I think it probably isn't needed if you don't need to add anything specific to it for your library, but the template includes it in case you do.